lib/ansible/modules/windows/win_uri.ps1

```diff
@@ -68,6 +67,10 @@ $JSON_CANDIDATES = @('text', 'json', 'javascript')
 $module.Result.elapsed = 0
 $module.Result.url = $url
 
+if (-not ($method -cmatch '^[A-Z]+$')) {
+    $module.FailJson("Parameter 'method' needs to be a single word in uppercase, like GET or POST.")
```

Comment:
we force 'upper' so we don't actually require it to be upercase on input

Reply:
Correct, the message is more strict than what we expect. Is that a problem ?
The message reflects correctly to what we test though.